src/components/App.tsx

```diff
@@ -1769,7 +1769,7 @@ export class App extends React.Component<any, AppState> {
     event.preventDefault();
     const { deltaX, deltaY } = event;
 
-    if (event.metaKey || event.ctrlKey) {
```

Comment:
@j-f1 any reason you moved away from platform-specific key to a union?

Reply:
I'll merge this. If this change was intentional, we can fix this later.

Reply:
Yes. Ctrl+scroll is what’s emitted by Chrome and Firefox when the user pinches on the trackpad.

Reply:
Pinch is handled by `wheel` event?

Reply:
Yes. Safari uses nonstandard `gesture*` events, but the other browsers use a `wheel` event.

Reply:
I see. Will make a fix.